test/parallel/test-cli-eval.js

```diff
@@ -113,3 +113,17 @@ child.exec(nodejs + ` -e 'require("child_process").fork("${emptyFile}")'`,
   assert.strictEqual(proc.stderr, '');
   assert.strictEqual(proc.stdout, 'start\nbeforeExit\nexit\n');
 }
+
+[ '-arg1',
+  '-arg1 arg2 --arg3',
+  '--',
+].forEach(function(args) {
+
+  child.exec(nodejs +
+             ' --eval "console.log(process.argv.slice(1).join(\' \'))" -- ' +
```

Comment:
If you do this string concatenation in a separate variable, this can be formatted in a much more readable fashion.